webapp-client/src/stylesheets/Item.css

```diff
@@ -0,0 +1,22 @@
+.item {
+  margin: 15px;
+  border-radius: 3px;
+  background-color: blanchedalmond;
+}
+
+.highlight {
+  font-size: 25px;
+}
+
+a {
+  font-size: 18px;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.notes {
```

Comment:
If you switch notes to comment in the other file, maybe switch this to comment as well?